### ISSUE TITLE
[ACCESSIBILITY] Assistant - No status message when Assistant is responding (TalkBack) 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4194]
**Describe the issue**
If a TalkBack (screen reader) user enters question, next they should be given a cue that the Assistant is processing or working on it. 

[WCAG 4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

To Reproduce

1. Using TalkBack
2. Go to 'Assistant'
3. Gesture until you hear 'Edit box, double tap to edit text'
4. Double tap, enter any question (for this I used 'Where is the nearest dining hall?' via voice entry button)

**Expected behavior**
After entering any question, screen reader user needs to be given a cue that the Assistant is processing or working on responding. 

**Actual behavior**
After entering any question, screen reader user hears "Cleared text, keyboard hidden, button, double tap to activate." They are not taken to a status message or response. 

**Screenshots**

https://github.com/rokwire/illinois-app/assets/82892873/dbb8d3f2-1583-4a16-bd9d-8e12edbd6cc4

**Testing Environment & Devices (please complete the following information):**
• OS: Android Version 13
• Device: Google Pixel 4a

**Additional context**

**Comment by TodorBachvarov** [comment]
@acovher2, I've made some improvements in version 6.0.20.

**Comment by acovher2** [comment]
It says " loading" which I think is enough indication to user something is processing and then it had focus land on and read the response. Verified as fixed in 6.0.36 android Prod
